subsys/fs/Kconfig.fatfs

```diff
@@ -129,6 +129,13 @@ config FS_FATFS_MAX_SS
 	range 512 4096
 	default 512
 
+config FS_FATFS_WIN_ALIGNED
+	int "Memory alignment for the member \"win\" in FATFS"
+	default 0
+	help
+	  Specify the alignment in byte eg: 16, 512. This is a low-level
+	  requirement by some of the mmc controller.
```

Comment:
The alignment happens inside FATFS structure which  means that the size of structure may significantly get bigger, when for example 512 alignment is selected. I think this should be noted here.

Reply:
Hi Ermel,

Okay will add the note to indicate that. Thanks.